### PR TITLE
Test OpenVINO export

### PR DESCRIPTION
**Pull request by adrianboguszewski**
## 📝 Description

@samet-akcay @ashwinvaidya17 is it the right place to test if all models are exportable to OpenVINO?

## ✨ Changes

Select what type of change your PR is:

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔨 Refactor (non-breaking change which refactors the code base)
- [x] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 📚 Documentation update
- [ ] 🔒 Security update

## ✅ Checklist

Before you submit your pull request, please make sure you have completed the following steps:

- [ ] 📋 I have summarized my changes in the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) and followed the guidelines for my type of change (skip for minor changes, documentation updates, and test enhancements).
- [ ] 📚 I have made the necessary updates to the documentation (if applicable).
- [ ] 🧪 I have written tests that support my changes and prove that my fix is effective or my feature works (if applicable).

### CODE REVIEW

File: tests/integration/model/test_models.py [hunk 1]
```diff
@@ -116,11 +121,19 @@ def test_predict(self, model_name: str, dataset_path: Path, project_path: Path)
         )
 
     @pytest.mark.parametrize("model_name", models())
-    def test_export(self, model_name: str, dataset_path: Path, project_path: Path) -> None:
+    @pytest.mark.parametrize("export_framework", frameworks())
+    def test_export(
+        self,
+        model_name: str,
+        export_framework: ExportType,
```
Comment (samet-akcay) [inline tests/integration/model/test_models.py:128]:
```suggestion
        export_type: ExportType,
```

File: tests/integration/model/test_models.py [hunk 2]
```diff
@@ -24,6 +24,11 @@ def models() -> list[str]:
     return get_available_models()
 
 
+def frameworks() -> list[ExportType]:
```
Comment (samet-akcay) [inline tests/integration/model/test_models.py:27]: how about the one below to follow the existing terminology?
```suggestion
def export_types() -> list[ExportType]:
```

File: tests/integration/model/test_models.py [hunk 3]
```diff
@@ -116,11 +121,19 @@ def test_predict(self, model_name: str, dataset_path: Path, project_path: Path)
         )
 
     @pytest.mark.parametrize("model_name", models())
-    def test_export(self, model_name: str, dataset_path: Path, project_path: Path) -> None:
+    @pytest.mark.parametrize("export_framework", frameworks())
```
Comment (samet-akcay) [inline tests/integration/model/test_models.py:124]:
```suggestion
    @pytest.mark.parametrize("export_type", export_types())
```

**Review by samet-akcay — CHANGES_REQUESTED** [review]
Thanks @adrianboguszewski. I have one suggestion to rename framework to export_type. Since framework term is not used in the main code base, the code might flow better if we stick to the `export_type` term that is used throughout the codebase

**Comment by adrianboguszewski** [comment]
@samet-akcay you review has been incorporated :)

**Review by samet-akcay — APPROVED** [review]

**Comment by nguyenanhtuan1008** [comment]
@adrianboguszewski @samet-akcay 
sorry, I have a question. I used command line for training: anomalib train --config <path/to/config>. Now I want to export the custom model to TORCH and OpenVINO model, how can I do it? 

This my command:
anomalib train --config /workspaces/anomalib/efficient_ad.yaml --data /workspaces/anomalib/mvtec.yaml 

mvtec.yaml:
```
model:
  class_path: anomalib.models.EfficientAd
  init_args:
    teacher_out_channels: 384
    model_size: M
    lr: 0.0001
    weight_decay: 1.0e-05
    padding: false
    pad_maps: true
    batch_size: 1

metrics:
  pixel:
    - AUROC

trainer:
  max_epochs: 1000
  max_steps: 70000

```

and efficient_ad.yaml:
```
model:
  class_path: anomalib.models.EfficientAd
  init_args:
    teacher_out_channels: 384
    model_size: M
    lr: 0.0001
    weight_decay: 1.0e-05
    padding: false
    pad_maps: true
    batch_size: 1

metrics:
  pixel:
    - AUROC

trainer:
  max_epochs: 1000
  max_steps: 70000

```

**Pull request merged**